Authentication/Login/View/LoginViewDelegate.swift

```diff
@@ -32,19 +32,19 @@ public extension LoginViewDelegate {
 public final class DefaultLoginViewDelegate: LoginViewDelegate {
     
     private let _configuration: LoginViewConfigurationType
-    
-    public let colourPalette: ColorPaletteType
+
+    public let colorPalette: ColorPaletteType
     public let fontPalette: FontPaletteType
```

Comment:
we can delete these properties and have them as computed properties from the configuration